specs/beans/request/legacy_target_converter.test.ts

```diff
@@ -134,7 +134,7 @@ describe("LegacyTargetConverter", () => {
                                 value: "HOURS"
                             }
                         ],
-                        enabled: false
+                        enabled: true
```

Comment:
Hmm... I'm guessing that the assertion on line 248 failed, and that prompted you to change this field?

Unfortunately, I think that that assertion _should fail_. If I look at the definition of `legacy_complex`, it specifies an `avg` filter with a sampling of `1h`. So, when that "legacy" object gets converted to a "modern" equivalent (i.e. this `modern_complex` object), the aggregator should _still_ aggregate every 1 hour -- which won't happen if the switch gets flipped to "auto value."

(As you may already know: if you ever find yourself changing an existing test, that's a big warning sign that you might be breaking something.)

As for how to avoid changing this test: it looks to me like [this](https://github.com/ArpNetworking/kairosdb-datasource/blob/cf04a45dfe8c161520b16ce97df04123b1fa3218/src/beans/request/legacy_target_converter.ts#L52) is the code that converts "legacy" aggregators to "modern" ones. You probably need to set the `enabled` flag there.